spec/src/main/asciidoc/ch04-query-language.adoc

```diff
@@ -834,27 +931,27 @@ WHERE p.status = 'inStock'
 GROUP BY s.name
 ----
 
-The result of the latter query will exclude
-suppliers who have no products in stock whereas the former query will
-include them.
+The result of the latter query will exclude suppliers who have no
+products in stock whereas the former query will include them.
 
-An important use case for LEFT JOIN is in
-enabling the prefetching of related data items as a side effect of a
-query. This is accomplished by specifying the LEFT JOIN as a FETCH JOIN
-as described below.
+An important use case for LEFT JOIN is in enabling the prefetching of
+related data items as a side effect of a query. This is accomplished by
+specifying the LEFT JOIN as a FETCH JOIN, as described below.
 
 ===== Fetch Joins [[a4931]]
 
-A FETCH JOIN enables the fetching of an
-association or element collection as a side effect of the execution of a
-query.
+A FETCH JOIN clause in a query results in eager fetching of an association
+or element collection as a side effect of execution of the query.
 
-The syntax for a fetch join is
+The syntax for a fetch join is given by:
 
 ----
 fetch_join ::= [LEFT [OUTER] | INNER] JOIN FETCH join_association_path_expression
```

Comment:
This grammar now differs from the BNF i.e. the BNF "allows" right and full join fetches as well. Maybe the BNF should be adapted to include a `fetch_join_spec`?

Reply:
> Maybe the BNF should be adapted to include a `fetch_join_spec`?

I wondered that, but then decided it wasn't necessary, since the text of section 4.4.5.3 says explicitly that right and full joins are disallowed.

But I'm not _against_ adding a separate rule.

Reply:
> But I'm not against adding a separate rule.

Just especially for you, I made the change.